docs/detections/session-view.asciidoc

```diff
@@ -84,4 +93,37 @@ Session View includes an additional badge not pictured above:
 * The *Exec user change* badge highlights exec user changes, such as when a user escalates to root:
 +
 [role="screenshot"]
-image::images/session-view-exec-user-change-badge.png[The Exec user change button,width=80%,height=80%]
+image::images/session-view-exec-user-change-badge.png[The Exec user change badge,width=80%,height=80%]
+
+* The *Output* badge appears next to commands which generated TTY output. Click it to open Terminal output view.
++
+[role="screenshot"]
+image::images/session-view-output-badge.png[The Output badge,width=80%,height=80%]
+
+[[session-view-output]]
+[discrete]
+=== Terminal output view UI
+
+beta::[]
+
+Terminal output view helps provide a more complete understanding of commands executed by users or adversaries, and assists with auditing and compliance objectives. It allows you to review a session's TTY output, in the context of the commands which generated it. It includes both TTY output, and shell commands (if the shell was in echo mode).
+
+To enable Terminal output data capture:
+. Go to *Manage* -> *Policies*, then select one or more of your {elastic-defend} integration policies to edit.
+. On the *Policy settings* tab, scroll down to the Linux event collection section near the bottom of the page
+and select the *Include session data* and *Capture terminal output* options.
+
+You can configure several additional settings by clicking *Advanced settings* at the bottom of the page:
+
+* Select the maximum number of kilobytes of output to record from a single process by modifying `linux.advanced.tty_io.max_kilobytes_per_process`. Default: 512 KB. For example, with the default value, a process' output will not be recorded after the 512th KB.
+* Select the maximum number of kilobytes of output to send to {es} as a single event by modifying `linux.advanced.tty_io.max_kilobytes_per_event`. Default: 512 KB. For example, with the default value, the maximum amount of output that will be sent to {es} as a single event is 512 KB. Additional data will be captured as a new event.
+* Select the maximum duration in seconds for which output is batched by modifying `linux.advanced.tty_io.max_event_interval_seconds`. Default: 30 seconds. For example, with the default value, any recorded output will be sent to {es} every 30 seconds (unless it first exceeds the `max_kilobytes_per_event` value, in which case it might be sent sooner).
+
+[role="screenshot"]
+image::images/session-view-output-viewer.png[Terminal output view,width=100%,height=100%]
+
+1. The search bar. Use to find and highlight search terms within the current session.
+The left and right arrows allow you to navigate through search results.
+2. The right-side scroll bar. Use along with the bottom scroll bar to navigate output data that doesn't fit on a single screen.
```

Comment:
This item feels really unnecessary: it sounds like these scroll bars work the same as every other scroll bar in every other app, so I don't know why we're calling attention to them. If we're concerned that users might not immediately realize they can scroll through output data, it's probably more appropriate to mention that as part of describing the general UI.

Reply:
This is a great point, hilariously stated, and I am making a note of it, though I'm not going to change it right now.

Reply:
There is a difference with the way the scroll area works in this context. It doesn't scroll the text content in the terminal, it is a scroll area to vertically/horizontally position the inner terminal window (which is statically sized to the native host session size, e.g 120x80 (character columns/rows). If a host session is large, e.g 200x120 or something, it will exceed the bounds of the container, and so the combination of this scroll area, the zoom in / out and the fullscreen controls allow you to size this area so that it's native session fits the scrollpane. You'll notice if you hit the "fit to screen" button (when not fullscreen) for a session that has a natively large size, it will shrink the text inorder for the window to fit in the smaller scroll area.

Reply:
Thanks for explaining, @mitodrummer! I definitely think it's worth mentioning how the user can scroll, resize, etc. in the UI, and some of these details would be helpful to mention. 

@benironside if there's a way to summarize that in the current description, I think that'd work and it would be clearer why we're saying "look, a scroll bar." If it's too much info to summarize easily, then it might justify a separate section on navigating terminal session sizes, etc.